command/aarelayer/service/config/config.json

```diff
@@ -0,0 +1,3 @@
+{
+    "whitelist": null
+}
```

Comment:
Missing LF at the end of the file, but why is this included in the PR?

Reply:
I thought it is just small functionality so i wanted to push it through this pr :). Is it ok if I just add LF at the end?

Reply:
`LF` on the end https://github.com/0xPolygon/polygon-edge/pull/1252/commits/f8527534f60da34f423360164a5bfd3a2cbf29d9